wagtail/admin/views/pages/move.py

```diff
@@ -106,5 +134,6 @@ def move_confirm(request, page_to_move_id, destination_id):
         {
             "page_to_move": page_to_move,
             "destination": destination,
+            "pages_to_move": len(pages_to_move),
```

Comment:
`pages_to_move` includes the source page and the alias pages. The message refers to just translated pages, so this would need to be something like


```suggestion
            "translations_to_move_count": pages_to_move.filter(alias_of__isnull=True).exclude(pk=page_to_move).count(),
```